packages/typespec-client-generator-core/src/decorators.ts

```diff
@@ -925,6 +925,13 @@ export function getUsageOverride(
   return getScopedDecoratorData(context, usageKey, entity);
 }
 
+export function getDefaultUsage(context: TCGCContext, namespace?: Namespace): UsageFlags {
+  if (!namespace) {
+    return UsageFlags.None;
+  }
+  return getScopedDecoratorData(context, usageKey, namespace) ?? UsageFlags.None;
+}
+
```

Comment:
why not merge this logic to `getUsageOverride`? i don't think it's kind of default usage, but kind of usage override on wide scale.

Reply:
good point, let me fix this